### YamlMime:JavaType
uid: "com.azure.resourcemanager.applicationinsights.models.ErrorDefinition"
fullName: "com.azure.resourcemanager.applicationinsights.models.ErrorDefinition"
name: "ErrorDefinition"
nameWithType: "ErrorDefinition"
summary: "Error response."
inheritances:
- "<xref href=\"java.lang.Object\" data-throw-if-not-resolved=\"False\" />"
- "<xref href=\"com.azure.core.management.exception.ManagementError\" data-throw-if-not-resolved=\"False\" />"
inheritedMembers:
- "com.azure.core.management.exception.ManagementError.getAdditionalInfo()"
- "com.azure.core.management.exception.ManagementError.getCode()"
- "com.azure.core.management.exception.ManagementError.getDetails()"
- "com.azure.core.management.exception.ManagementError.getMessage()"
- "com.azure.core.management.exception.ManagementError.getTarget()"
- "com.azure.core.management.exception.ManagementError.toString()"
- "java.lang.Object.clone()"
- "java.lang.Object.equals(java.lang.Object)"
- "java.lang.Object.finalize()"
- "java.lang.Object.getClass()"
- "java.lang.Object.hashCode()"
- "java.lang.Object.notify()"
- "java.lang.Object.notifyAll()"
- "java.lang.Object.wait()"
- "java.lang.Object.wait(long)"
- "java.lang.Object.wait(long,int)"
syntax: "public final class ErrorDefinition extends ManagementError"
constructors:
- "com.azure.resourcemanager.applicationinsights.models.ErrorDefinition.ErrorDefinition()"
methods:
- "com.azure.resourcemanager.applicationinsights.models.ErrorDefinition.getInnererror()"
- "com.azure.resourcemanager.applicationinsights.models.ErrorDefinition.validate()"
type: "class"
metadata: {}
package: "com.azure.resourcemanager.applicationinsights.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-applicationinsights:1.0.0-beta.1
